onion_buildenv

```diff
@@ -90,8 +90,15 @@ download_builder() {
 	return 0
 }
 
+clean_additions() {
+	if [ -n "$IMAGE_BUILDER_USER_FILES" ] && [ -d "$IMAGE_BUILDER_USER_FILES" ]; then
+		rm -rf "$IMAGE_BUILDER_USER_FILES" || return 1
```

Comment:
Can we add a print statement to let the user know we've deleted the existing `files` directory in the image builder?